.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,17 +2,17 @@
 name: Feature request
 about: Suggest an idea for this project
 title: ''
-labels: enhancement
+labels: ''
```

Comment:
What's the reasoning behind changing this? I'm used to automatically add the `enhancement` label for features, which seems right. I also wonder if the empty string is the right way to specify `null` or empty list.

Reply:
Not sure, I edited in the in the UI, not sure why it removed this.